code/modules/research/designs/weapon_designs.dm

```diff
@@ -362,3 +362,13 @@
 	build_path = /obj/item/melee/baton/security/boomerang
 	category = list("Weapons")
 	departmental_flags = DEPARTMENTAL_FLAG_SECURITY
+
+/datum/design/pie_cannon
+	name = "Banana Cream Pie Cannon"
+	desc = "Load cream pie for optimal results."
+	id = "pie_cannon"
+	build_type = PROTOLATHE | AWAY_LATHE
+	materials = list(/datum/material/iron = 10000, /datum/material/bananium = 3000)
```

Comment:
requires bananium, sad

Reply:
Clowns stay mad, they have to work for a second pie cannon